src/components/EditItemsList/EditItemsList.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { Wrap, Input, Box, Button, Flex } from '@chakra-ui/react';
+import ItemCard from '../ItemCard/ItemCard';
+
+const EditItemsList = ({ items, setItems, isAccepted = false }) => {
```

Comment:
🚫 **[eslint]** <[react/prop-types](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'setItems' is missing in props validation